src/route/healthcheck.ts

```diff
@@ -1,25 +1,31 @@
 import { Router } from 'express';
+import passport from 'passport';
 
-// eslint-disable-next-line import/no-cycle
-import { logger } from '../app';
-import { DataLakeService } from '../controllers/datalake';
+// import { logger } from '../utils/logger';
+// import { DataLakeService } from '../controllers/datalake';
+import { sanitiseUser } from '../utils/sanitise-user';
+import { User } from '../entities/user';
 
-const dataLakeService = new DataLakeService();
-export const healthcheck = Router();
+const healthcheck = Router();
 
```

Comment:
We do need a complete `/` healthcheck to pass to Docker and the Container service to tell us when things aren't working.  Really this should include checking we have the following:
- Database connection
- Datalake connection
- BlobStorage connection

As these are required for the service to work and if they stop working for any reason its good to be able to look at the health check to see thats the case.

Reply:
I agree, we should have a healthcheck/<service> for any connected services that we can use for monitoring.

I had to disable the datalake one temporarily because it is failing in CI when attempting to connect to `your-storage-account-name.x.y.z` endpoint to fetch creds.